examples/CopyWorkspaceSampleScript.ps1

```diff
@@ -137,7 +137,12 @@ Foreach ($report in $reports) {
 
     Write-Host "== Exporting $report_name with id: $report_id to $temp_path"
     try {
-        Export-PowerBIReport -Id $report_id -OutFile "$temp_path" -ErrorAction Stop
+        if ($source_workspace_ID -eq "me") {
```

Comment:
I had to make one tweak to make the script work with personal workspace ("My Workspace"). Thanks for fixing this issue!